src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -55,7 +55,7 @@ public function __construct($queryBuilder, $manager = null, $class = null)
 
             $queryBuilder = $queryBuilder($manager->getRepository($class));
 
-            if (!$queryBuilder instanceof QueryBuilder) {
+            if (null !== $queryBuilder && !$queryBuilder instanceof QueryBuilder) {
```

Comment:
If I remove this condition, I will get the UnexpectedTypeException if the Closure returns null. So we need to let ``null`` as supported value here.